deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -841,80 +476,426 @@ collectors:
               - resourcedetection/aks
               - resource/k8s
               - resource/hostname
               - resource/cloud
-              - attributes/dataset
-              - resource/process
             exporters:
-              - debug
-              - elasticsearch/ecs
+              # - debug
+              - otlp/gateway
           metrics/otel-apm:
             receivers:
               - otlp
             processors:
               - batch
               - resource/hostname
             exporters:
-              - debug
-              - signaltometrics
-              - elasticsearch/otel
+              - otlp/gateway
           logs/apm:
             receivers:
               - otlp
             processors:
               - batch
               - resource/hostname
             exporters:
-              - debug
-              - signaltometrics
-              - elasticsearch/otel
+              - otlp/gateway
           traces/apm:
             receivers:
               - otlp
             processors:
               - batch
-              - elastictrace
               - resource/hostname
             exporters:
-              - debug
-              - signaltometrics
-              - elasticsearch/otel
+              - otlp/gateway
+  #  Gateway is a K8s deployment EDOT collector focused on processing and
+  #  forwarding telemetry to an Elasticsearch endpoint.
+  gateway:
+    resources:
+      limits:
+        cpu: 1500m
+        memory: 1500Mi
+      requests:
+        cpu: 100m
+        memory: 500Mi
+    suffix: gateway
+    replicas: 2
+    enabled: true
+    env:
+      - name: ELASTIC_AGENT_OTEL
+        value: '"true"'
+      - name: ELASTIC_ENDPOINT
+        valueFrom:
+          secretKeyRef:
+            name: elastic-secret-otel
+            key: elastic_endpoint
+      - name: ELASTIC_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: elastic-secret-otel
+            key: elastic_api_key
+      - name: GOMAXPROCS
+        valueFrom:
+          resourceFieldRef:
+            resource: limits.cpu
+      - name: GOMEMLIMIT
+        value: "1025MiB"
+    config:
+      connectors:
+        routing:
+          default_pipelines: [metrics/otel]
+          error_mode: ignore
+          match_once: true
+          table:
+            - context: metric
+              statement: route() where instrumentation_scope.name == "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver" or IsMatch(instrumentation_scope.name, "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/*")
+              pipelines: [metrics/infra/ecs, metrics/otel]
+        # [Signal To Metrics Connector](https://github.com/elastic/opentelemetry-collector-components/tree/main/connector/signaltometricsconnector)
+        signaltometrics: # Produces metrics from all signal types (traces, logs, or metrics).
+          logs:
```

Comment:
shall we inlcude the orchestrator.cluster.name ? 
It is something that in general needed to filter with.

I added the comment in logs but same for metrics as well